CONTRIBUTING.md

```diff
@@ -0,0 +1,228 @@
+# Contributing
+
+You want to contribute and add tests to this repository? Great! We are happy to
+accept your contributions. Please read the following guidelines before you
+start.
+
+## Guidelines for contributing
+
+**TL;DR**
+
+* PR title and description should use a prefix like `[W09H03] Add behavioral tests`
+* Separate your tests logically into small files
+* Don't add tests that are not needed for the exercise
+* Write deterministic tests
+* Use JUnit 5.9.1
+* Format your code
+* Don't depend on changing the code of the implementation
+* Every test file should end with `Test.java`
+* Follow the package structure of the implementation
+* Only one test case per test method
+* Use descriptive names for test methods.
+
+### PR Title and Description
+
+#### Title
+
+Please as prefix for your PR title the name of the exercise you are working on.
+
+```
+[W09H03] Add behavioral tests
+```
+
+If you don't changed the tests for an exercise, you can use `[*]` as prefix:
+
+```
+[*] ...
+```
+
+#### Description
+
+In the description of your PR, please add a description that helps us to understand what you are doing. For example, when you fix a bug, please add a description of the bug and how you fixed it.
```

Comment:
```suggestion
The description of your PR should help us understand what you are doing: For example, when fixing a bug, provide information on the bug and a brief sentence on how you fixed it.
```
not good to repeat long words in the same sentence imo

Reply:
Updated ✅